elements/lisk-transactions/src/15_proof_of_misbehavior_transaction.ts

```diff
@@ -0,0 +1,172 @@
+/*
+ * Copyright © 2020 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import {
+	getAddressFromPublicKey,
+	intToBuffer,
+} from '@liskhq/lisk-cryptography';
+import { isNumberString, validator } from '@liskhq/lisk-validator';
+
+import {
+	BaseTransaction,
+	StateStore,
+	StateStorePrepare,
+} from './base_transaction';
+import { convertToAssetError, TransactionError } from './errors';
+import { Account, BlockHeader, TransactionJSON } from './transaction_types';
+import { getBlockBytes } from './utils';
+
+const proofOfMisbehaviorAssetFormatSchema = {
+	type: 'object',
+	required: ['header1', 'header2'],
+	properties: {
+		header1: {
```

Comment:
maybe be extract this `header1` json schema into a variable 
ex:
const blockHeaderSchema = { key: value }
and then
```
properties: {
  header1: ...blockHeaderSchema,
  header2: ...blockHeaderSchema,
}
?